ansible_ai_connect/main/settings/base.py

```diff
@@ -36,51 +36,47 @@
 # Quick-start development settings - unsuitable for production
 # See https://docs.djangoproject.com/en/4.1/howto/deployment/checklist/
 
-ANSIBLE_AI_MODEL_MESH_HOST = os.getenv(
-    "ANSIBLE_AI_MODEL_MESH_HOST", "https://model.wisdom.testing.ansible.com"
-)
-ANSIBLE_AI_MODEL_MESH_INFERENCE_PORT = os.getenv("ANSIBLE_AI_MODEL_MESH_INFERENCE_PORT", 443)
-ANSIBLE_AI_MODEL_MESH_INFERENCE_URL = (
-    f"{ANSIBLE_AI_MODEL_MESH_HOST}:{ANSIBLE_AI_MODEL_MESH_INFERENCE_PORT}"
-)
-
+# ==========================================
+# Model Provider
+# ------------------------------------------
 t_model_mesh_api_type = Literal[
     "grpc", "http", "dummy", "wca", "wca-onprem", "wca-dummy", "ollama", "llamacpp", "bam"
 ]
 ANSIBLE_AI_MODEL_MESH_API_TYPE: t_model_mesh_api_type = os.getenv(
     "ANSIBLE_AI_MODEL_MESH_API_TYPE"
 ) or cast(t_model_mesh_api_type, "http")
-t_model_mesh_healthcheck_protocol = Literal["http", "https"]
-ANSIBLE_AI_MODEL_MESH_API_HEALTHCHECK_PROTOCOL: t_model_mesh_healthcheck_protocol = os.getenv(
-    "ANSIBLE_AI_MODEL_MESH_API_HEALTHCHECK_PROTOCOL"
-) or cast(t_model_mesh_healthcheck_protocol, "https")
-ANSIBLE_AI_MODEL_MESH_API_HEALTHCHECK_PORT = (
-    ANSIBLE_AI_MODEL_MESH_INFERENCE_PORT
-    if ANSIBLE_AI_MODEL_MESH_API_TYPE == "http"
-    else (
-        os.getenv("ANSIBLE_AI_MODEL_MESH_API_HEALTHCHECK_PORT", "8443")
-        if ANSIBLE_AI_MODEL_MESH_API_TYPE == "grpc"
-        else None
-    )
-)
-
-ANSIBLE_AI_MODEL_NAME = os.getenv("ANSIBLE_AI_MODEL_NAME") or "wisdom"
 
+t_model_mesh_protocol = Literal["http", "https"]
+ANSIBLE_AI_MODEL_MESH_PROTOCOL: t_model_mesh_protocol = os.getenv(
+    "ANSIBLE_AI_MODEL_MESH_PROTOCOL"
+) or cast(t_model_mesh_protocol, "https")
+ANSIBLE_AI_MODEL_MESH_HOST = (
+    os.getenv("ANSIBLE_AI_MODEL_MESH_HOST") or "https://model.wisdom.testing.ansible.com"
+)
+ANSIBLE_AI_MODEL_MESH_INFERENCE_PORT = int(
+    os.getenv("ANSIBLE_AI_MODEL_MESH_INFERENCE_PORT") or "443"
+)
+ANSIBLE_AI_MODEL_MESH_INFERENCE_URL = os.getenv("ANSIBLE_AI_MODEL_MESH_INFERENCE_URL") or (
+    f"{ANSIBLE_AI_MODEL_MESH_PROTOCOL}://"
+    f"{ANSIBLE_AI_MODEL_MESH_HOST}:"
+    f"{ANSIBLE_AI_MODEL_MESH_INFERENCE_PORT}"
+)
 ANSIBLE_AI_MODEL_MESH_API_KEY = os.getenv("ANSIBLE_AI_MODEL_MESH_API_KEY")
-ANSIBLE_AI_MODEL_MESH_MODEL_NAME = os.getenv("ANSIBLE_AI_MODEL_MESH_MODEL_NAME")
```

Comment:
It could be nice to rice a warning or fail if `os.getenv("ANSIBLE_AI_MODEL_MESH_MODEL_NAME")` is `True`.

Reply:
Done.